src/main/groovy/jenkins/automation/builders/MultibranchPipelineJobBuilder.groovy

```diff
@@ -1,55 +1,85 @@
 package jenkins.automation.builders
 
-import jenkins.automation.utils.CommonUtils
-import jenkins.automation.utils.Branch
-
 import javaposse.jobdsl.dsl.DslFactory
 import javaposse.jobdsl.dsl.jobs.MultibranchWorkflowJob
-import jenkins.automation.utils.Branch
+
+
+/*
+    NB: if you are polling via the GitHub API (github.com or GHE) and
+        want to use webhooks with a multi-branch pipeline, but the webhooks
```

Comment:
'multi-branch' or "multibranch?' Either way it will help to be consistent.

Reply:
good call; no hyphen I believe is the correct one, will fix